pkg/assembler/helpers/vcs.go

```diff
@@ -66,6 +65,10 @@ func VcsToSrc(vcsUri string) (*model.SourceInputSpec, error) {
 	} else {
 		m.Name = strings.TrimPrefix(u.Path, "/")
 	}
+	// Based on the issue https://github.com/guacsec/guac/issues/1413, we need to ensure
+	// that the .git suffix is removed from the repository name. This is because some
+	// repository URLs might include the .git suffix which is not expected by certain endpoints.
+	m.Name = strings.TrimSuffix(m.Name, ".git")
```

Comment:
Should we also trim `.git` after possibly removing the version? If there is a version specified after the `.git` then it won't be removed.

Reply:
Yeah, the code now trims the `.git` after removing the version. Thanks!